bootstrap/templates/new/mix.exs

```diff
@@ -12,14 +12,14 @@ defmodule <%= app_module %>.Mixfile do
      version: "0.1.0",
      elixir: "<%= elixir_req %>",
      target: @target,
-     archives: [nerves_bootstrap: "~> <%= bootstrap_vsn %>"],<%= if in_umbrella do %>
-     deps_path: "../../deps/#{@target}",
+     archives: [nerves_bootstrap: "~> <%= bootstrap_vsn %>"],
+<%= if in_umbrella do %>     deps_path: "../../deps/#{@target}",
```

Comment:
While this isn't great for readability in the template itself, I think that creating clean output is more important.